tasks/main.yml

```diff
@@ -351,7 +351,7 @@
     no_proxy: "{{ vault_address }}"
   uri:
     validate_certs: "{{ validate_certs_during_api_reachable_check | bool }}"
-    url: "{{ vault_addr_protocol }}://{{ vault_hostname | default(vault_addr, true) }}:{{ vault_port }}/v1/sys/health"
+    url: "{{ vault_api_addr }}/v1/sys/health"
```

Comment:
Why this change? Where is vault_api_addr defined?

Reply:
right here sir:

```yaml
vault_api_addr: "{{ vault_protocol }}://{{ vault_redirect_address | default(hostvars[inventory_hostname]['ansible_'+vault_iface]['ipv4']['address']) }}:{{ vault_port }}"
```
https://github.com/ansible-community/ansible-vault/blob/master/defaults/main.yml#L125